pages/mi/mi-myplanet-and-android-studio.md

```diff
@@ -20,11 +20,19 @@ After importing the project, Android Studio will perform a Gradle build, which m
 
 ## 4. Configure Emulator
 
-To avoid overriding the app on your physical device and to ensure crash reports are sent, we need to set up an Android emulator for this step. Open 'Tools' > 'Device Manager.' Click the + icon to create a new virtual device and follow the prompts.
+To avoid overriding the app on your physical device and to ensure crash reports are sent, we need to set up an Android emulator for this step. Open 'Tools' > 'Device Manager.'   
+![image](https://github.com/strawberrybread/strawberrybread.github.io/assets/112584985/599aa357-50cc-4053-a729-b3572ae41cfa)
```

Comment:
We typically add the screenshot to `pages/mi/image/` folder and give it a somewhat descriptive name. Please add the images to this folder because we are not sure how long GitHub retains those media uploads, some online search mentions 30 days.

Reply:
Images have been added to `pages/mi/image/`

Reply:
Please take a look at https://github.com/open-learning-exchange/open-learning-exchange.github.io/tree/master/pages/mi/image to see how image naming conventions are currently. If you could do `git mv pages/mi/image/Device_Manager_location.png pages/mi/image/mi-android-studio-device-manager.png` and likewise for the other two images it would be great!

Afterwards, could you update the markdown file to use those new uploaded images as well?

Reply:
I was having trouble navigating my directories with git bash and I finally just realized it was because I forgot to sync my repo.